core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequestBuilder.java

```diff
@@ -84,4 +84,16 @@ public GetSnapshotsRequestBuilder addSnapshots(String... snapshots) {
         request.snapshots(ArrayUtils.concat(request.snapshots(), snapshots));
         return this;
     }
+
+    /**
+     * Makes the request to  to ignore unavailable snapshots,
+     *
+     * @param ignoreUnavailable true to ignore unavailable snapshots.
```

Comment:
Same as in GetSnapshotRequest, replace this by "Set to true to ignore unavailable snapshots"